On 10.5 the build fails with the error:
gcc-mp-7: error: unrecognized command line option '-stdlib=libc++'

see Trac ticket: https://trac.macports.org/ticket/61752

--- setup.py~	2020-11-02 03:14:28.000000000 +0100
+++ setup.py	2020-12-06 02:15:16.000000000 +0100
@@ -54,9 +54,6 @@
         for ext in self.extensions:
             ext.include_dirs.insert(0, cppy.get_include())
             ext.extra_compile_args = opts
-            if sys.platform == 'darwin':
-                ext.extra_compile_args += ['-stdlib=libc++']
-                ext.extra_link_args += ['-stdlib=libc++']
             if (ct == 'msvc' and os.environ.get('KIWI_DISABLE_FH4')):
                 # Disable FH4 Exception Handling implementation so that we don't
                 # require VCRUNTIME140_1.dll. For more details, see:
